crates/meroctl/src/cli.rs

```diff
@@ -54,6 +56,8 @@ pub enum SubCommands {
     Run(RunCommand),
     Context(ContextCommand),
     App(AppCommand),
+    #[command(alias = "call")]
+    JsonRpc(JsonRpcCommand),
```

Comment:
Anywhere we add stuff to lists, ideally we should do so in alphabetical order 🙂 Makes things much nicer over time.

Reply:
https://github.com/calimero-network/core/pull/696#discussion_r1746859497